src/app/best_tip_merger/best_tip_merger.ml

```diff
@@ -205,7 +205,7 @@ module Compact_display = struct
         { current : State_hash.t
         ; parent : State_hash.t
         ; blockchain_length : Mina_numbers.Length.t
-        ; global_slot : Mina_numbers.Global_slot.t
+        ; global_slot : Mina_numbers.Global_slot_since_genesis.t
```

Comment:
`Global_slot_since_hard_fork` but should be `Global_slot_since_genesis` but the logic needs to be updated before changing the type. opened #13184

Reply:
I will leave it be for now.